rpa-builder/6.0/modules/ROOT/pages/toolbox-checks-image-search.adoc

```diff
@@ -1,181 +1,239 @@
+desktop enclosing the image to be searched.
+* *Percentage required*
++
+Determines tolerance of the _Image search_, i.e.
 the minimum quality percentage threshold needed for successful
 identification of an image. A lower value means a higher tolerance, but
-also increases the likelihood of a "false positive". We *strongly*
-recommend never setting this below 90%!
-* *Relative To* Makes the _Search Area_ relative to a previous _Image
-search_, in the same way as for the _Mouse Clicks Action Step_
-* *_Search area_ rect* (_Properties grid_ only): Original coordinates
+also increases the likelihood of a false positive. To avoid false positives, do not set this value below 90%.
+* *Relative To*
++
+Makes the _Search Area_ relative to a previous _Image
+search_, in the same way as for the _Mouse Clicks Action Step_.
+* *_Search area_ rect*
++
+(_Properties grid_ only): Original coordinates
 (upper left corner, lower right corner) of the rectangular area on the
-desktop enclosing the _Search area_
-* *Wait time [ms]* The pause between two subsequent __Image search__es
-(only used if _Wait until image is steady_ is activated)
-* *Wait until image is steady* This function is used to perform a second
+desktop enclosing the _Search area_.
+* *Wait time [ms]*
++
+The pause between two subsequent __Image search__es
+(only used if _Wait until image is steady_ is activated).
+* *Wait until image is steady*
++
+This function is used to perform a second
 _Image search_ after the image is detected and introduces an additional
 _Wait time._ The result of this second run compared to the previous one
 is to ensure that the image is still there and has not changed its
 position. If the results are equal, then the _Image search_ is
 considered successful. If not, additional __Image search__es are
 performed until two subsequent ones have the same result or until the
 timeout is reached.
-* *Interrupt time [ms]* Mainly used for _Java_ Applications: time in
+* *Interrupt time [ms]*
++
+Mainly used for _Java_ Applications: time in
 milliseconds the service stops searching for a picture to
 provide CPU resources to an application with bad performance (only
-applicable if _Load Optimizer_ is activated)
-* *Load Optimizer* If enabled, there is a pause between every attempt to
+applicable if _Load Optimizer_ is activated).
+* *Load Optimizer*
++
+If enabled, there is a pause between every attempt to
 find the image onscreen, as defined by the _Interrupt time_ setting.
 This pause allows time for other programs to do something when running
 on a single core computer.
-* *Performance Limit (%)* Defines how much of the CPU performance can be
+* *Performance Limit (%)*
++
+Defines how much of the CPU performance can be
 used for the _Image search_ (only applicable if _Load Optimizer_ is
-activated)
+activated).
 
 == Variables
 
-* *Find Coordinate* The X/Y coordinate of where the picture was found
+* *Find Coordinate*
++
+The X/Y coordinate of where the picture was found.

```

Comment:
Would it be better to say "X and Y" and "coordinates"? A Google search showed that the most common expression is "X and Y coordinates".